pkg/process/net/windows_pipe.go

```diff
@@ -39,16 +42,22 @@ func newPipeListener(namedPipeName string) (net.Listener, error) {
 	return winio.ListenPipe(namedPipeName, &config)
 }
 
+// OverrideSystemProbeNamedPipePath sets the active named pipe path for System Probe connections.
+// This is used by tests only to avoid conflicts with an existing locally installed Datadog agent.
```

Comment:
You can enforce test-only by putting this function in a file with a build tag of `test`